test/test_block_token.py

```diff
@@ -377,6 +377,22 @@ def test_footnotes_may_not_have_too_much_leading_space(self):
         self.assertIsInstance(token.children[2], block_token.BlockCode)
         self.assertEqual(token.children[2].children[0].content, "[i-am-block-too]: /foo\n")
 
+    def test_footnotes_angle_bracketed_link_with_space(self):
```

Comment:
I can see you've taken this right from the spec, for simplicity. Yet, for better readability within existing tests, I would suggest to move it after `test_parse_with_title()` and to simplify like this:

```python
    def test_parse_with_space_in_every_part(self):
        lines = ['[foo bar]:\n',
                 '<my url>\n',
                 '\'my title\'\n']
        token = block_token.Document(lines)
        self.assertEqual(token.footnotes, {"foo bar": ("my url", "my title")})
```

(no need to import `core_tokens` with this change then)

Reply:
ok

Reply:
thanks